src/reducers/exportSpaceReducer.js

```diff
@@ -0,0 +1,43 @@
+import { Map, List } from 'immutable';
+import {
+  FLAG_EXPORTING_SPACE,
+  CLEAR_EXPORT_SPACE,
+  SET_EXPORT_SPACE,
+  FLAG_SETTING_EXPORT_SPACE,
+  EXPORT_SPACE_SUCCESS,
+  FLAG_CLEARING_EXPORT_SPACE,
+} from '../types';
+import { updateActivityList } from './common';
+
+export const EXPORT_SPACE_STATUS = {
+  IDLE: 'idle',
```

Comment:
What does `idle` mean?

Reply:
It means it does nothing. It opposes to `pending` where the action is processing. Should I change it to something else?